composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "tdiluccio/pretty-types",
+    "description": "Doctrine Type storing pretty printed JSON into database",
+    "keywords": ["Doctrine", "Json", "pretty"],
+    "require": {
+        "doctrine/dbal": "~2.5"
```

Comment:
`JsonType` is available only with `doctrine/dbal` ^2.6 (but JsonArray can still be used with 2.5).

JsonArrayTypes is 2.5 compatible but, shouldn't we force doctrine dbal 2.6 ? (I checked the changelog and it seems that nothing is breaking)

And if so, shouldn't we drop JsonArrayTypes as it's deprecated since 2.6 ?

Reply:
We should propably do that. And then upgrading doctrine/dbal in our other project as we massively use json_array.